zerver/openapi/zulip.yaml

```diff
@@ -9225,6 +9225,18 @@ paths:
                           array will be empty if `enable_drafts_synchronization` is set to `false`.
                         items:
                           $ref: "#/components/schemas/Draft"
+                      emoji_data_url:
+                        type: string
+                        description: |
+                          The URL to a JSON file that describes which emoji names map to which
+                          emoji codes, for all Unicode emoji this Zulip server accepts.
+
+                          The data at the given URL is a JSON object with one property, `name_to_code`.
+                          The value of that property is a JSON object where each key is an
+                          [emoji name](/api/add-reaction#parameter-emoji_name) for an available
+                          Unicode emoji, and each value is the corresponding
+                          [emoji code](/api/add-reaction#parameter-emoji_code).
+                          A given emoji code may appear under multiple emoji names.
```

Comment:
This API change needs to be documented with a `**Changes**` entry and API_FEATURE_LEVEL bump (see `version.py` for documentation).